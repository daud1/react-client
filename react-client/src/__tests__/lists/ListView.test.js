import React from 'react';
import { shallow, render, mount } from 'enzyme';
import ListView from '../../components/lists/ListView.component';
import shallowToJson from 'enzyme-to-json';

describe ('ListView Component', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<ListView />);
        expect(wrapper.exists(<div></div>)).toBe(true)
    });

    it('matches snapshot', () => {
        const wrapper = shallow(<ListView />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
})