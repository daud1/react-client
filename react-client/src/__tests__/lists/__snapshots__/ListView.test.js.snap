// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListView Component matches snapshot 1`] = `
<div>
  <Search
    setValue={[Function]}
  />
  <withRouter(LogOut) />
  <CreateList
    callback={[Function]}
  />
  <p>
     See all your lists here: 
  </p>
  <Lister
    callback={[Function]}
  />
</div>
`;
